tests/components/camera/test_init.py

```diff
@@ -238,38 +249,35 @@ def test_snapshot_service(hass, mock_camera):
 
     # Request preferences through websocket
     client = await hass_ws_client(hass)
-    await client.send_json(
-        {"id": 7, "type": "camera/get_prefs", "entity_id": "camera.demo_camera"}
-    )
+    await client.send_json({
+        "id": 7,
+        "type": "camera/get_prefs",
+        "entity_id": "camera.demo_camera"
+    })
     msg = await client.receive_json()
 
     # Assert WebSocket response
     assert msg["success"]
 
 
-async def test_websocket_update_prefs(
-    hass, hass_ws_client, mock_camera, setup_camera_prefs
-):
+async def test_websocket_update_prefs(hass, hass_ws_client, mock_camera,
+                                      setup_camera_prefs):
     """Test updating preference."""
     await async_setup_component(hass, "camera", {})
     assert setup_camera_prefs[PREF_PRELOAD_STREAM]
     client = await hass_ws_client(hass)
-    await client.send_json(
-        {
-            "id": 8,
-            "type": "camera/update_prefs",
-            "entity_id": "camera.demo_camera",
-            "preload_stream": False,
-        }
-    )
+    await client.send_json({
+        "id": 8,
+        "type": "camera/update_prefs",
+        "entity_id": "camera.demo_camera",
+        "preload_stream": False,
+    })
     response = await client.receive_json()
 
     assert response["success"]
     assert not setup_camera_prefs[PREF_PRELOAD_STREAM]
-    assert (
-        response["result"][PREF_PRELOAD_STREAM]
-        == setup_camera_prefs[PREF_PRELOAD_STREAM]
-    )
+    assert (response["result"][PREF_PRELOAD_STREAM] ==
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4639533)